x-pack/legacy/plugins/canvas/public/components/workpad_header/workpad_export/utils.ts

```diff
@@ -71,11 +71,11 @@ function getPdfUrlParts(
 
 export function getPdfUrl(...args: Arguments): string {
   const urlParts = getPdfUrlParts(...args);
+  const param = (key: string, val: any) =>
```

Comment:
Why not add the param function to the URL utils?

Reply:
We use this method only in one place. Not sure that we need it somewhere else